app/controllers/entries_controller.rb

```diff
@@ -7,16 +7,18 @@ def index
   end
 
   def show
-    @entry = Entry.find_by_id(entry_params[:id])
```

Comment:
Not sure why you changed it from entry_params to params.  Entry_params should still be good.